build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 
 allprojects {
   group = "uk.gov.justice.service.hmpps"
-  version = "1.1.8-wip"
+  version = "1.1.8"
```

Comment:
Although this is a breaking change the old `queue-admin-async` endpoints are only being used by a single client (hmpps-prisoner-from-nomis-migration) in dev and preprod in a job triggered by users. Therefore we can manage the version upgrade manually, hence this won't braek anything.